cli/cmd/upgrade.go

```diff
@@ -251,6 +251,13 @@ func upgrade(ctx context.Context, k *k8s.KubernetesAPI, flags []flag.Flag, stage
 			return bytes.Buffer{}, err
 		}
 	}
+	// if there is still no linkerd-config-overrides secret, Linkerd has been installed using
+	// Helm installation, where linkerd-config-overrides secret doesn't exist
+	// User must be notified to upgrade Linkerd using Helm
```

Comment:
If you can update this comment to reflect the new error message, that'd be great!

Reply:
Oops! I forgot to replace that. Thanks for notifying me about this :smiley: 
I replaced the comment with

`// if we fail to fetch linkerd-config-overrides secret;
	// Linkerd maybe installed using Helm installation,
	// where linkerd-config-overrides secret doesn't exist`